prime-router/settings/organizations.yml.orig

```diff
@@ -0,0 +1,2543 @@
+---
```

Comment:
.orig files should (almost) never be checked in

Reply:
This one filtered through after the merge. I just deleted it as well.